### ISSUE TITLE
Visualization of synthetic images in CIFAR10

### ISSUE THREAD

**Issue by horseee** [princetonvisualai/RememberThePast-DatasetDistillation#2]
Hi.

How to visualize the synthetic images if I activate ZCA pre-processing (as in Figure 7-9 in your paper)? 

I tried to visualize the images stored in the weight in the module nn.Embedding using BPTT without memory addressing, but it appears to be an almost black image. Is this a normal case for the BPTT+momentum algorithm? I got ~49% accuracy on ipc1 in CIFAR10, which is a close value to the reported result in your paper. So I guess maybe I miss something about how to visualize the images. Perhaps in the case of ZCA?

Thanks.

**Comment by horseee** [comment]
I solve this problem. Thanks!
